frontend/public/components/_nav.scss

```diff
@@ -3,7 +3,8 @@ $link-padding-left: 40px;
 $section-padding-left: 13px;
 $active-border: 3px;
 
-.co-m-nav-link {
+.co-m-nav-link,
+.co-m-masthead-link {
```

Comment:
If you pull out these styles as @rhamilto recommends, you should be able to get rid of the `!important` override above.

```css
.co-m-nav-link {
  &:hover {
   background-color: $color-os-nav-background-active;
  }

  &:focus {
   background-color: transparent;
  }
}
```

See https://github.com/openshift/console/pull/540#discussion_r221079268

Reply:
I am trying to make the hover behavior of the Import YAML link match that of the namespace selector. This behavior is different than the hover behavior of the side bar nav items. That is why the !important override is there.

Reply:
But if you break up the rules as @rhamilto suggests, you shouldn't need `!important` because you won't have to override a value.